DESCRIPTION

```diff
@@ -25,7 +25,7 @@ BugReports: https://github.com/easystats/datawizard/issues
 Depends:
     R (>= 3.4)
 Imports:
-    insight (>= 0.16.0.5),
+    insight (>= 0.16.9.0),
```

Comment:
0.17.0 is now on CRAN

Reply:
I can't work on this during office hours. 

Can you please make this change yourself and let me know? I can then submit it to CRAN.